index.html

```diff
@@ -1133,10 +1189,17 @@ <h2>Services</h2>
 
       <pre class="example nohighlight" title="Usage of the service property">
 {
-  "service": [{
+  "id": "https://controller.example",
+  "authentication": [{
+      "id": "https://controller.example#authn-key-123",
+      "type": "Multikey",
+      "controller": "https://controller.example",
+      "publicKeyMultibase": "z6MkmM42vxfqZQsv4ehtTjFFxQ4sQKS2w6WR7emozFAn5cxu"
+  }],
+  <span class="highlight">"service": [{
     "type": "ExampleSocialMediaService",
```

Comment:
I'm not sure this is a good example since the type isn't fully qualified, which means it defaults to being scoped to *this* controlled identifier document -- which is probably a useless type for any interoperability purposes. I suggest fully qualifying it instead:

```suggestion
    "type": "https://social.example/ExampleSocialMediaService",
```